### DISCUSSION TITLE
Fresh any table content after modal action

### DISCUSSION THREAD

**Discussion by indigoram89** [filamentphp/filament#13481] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Hello guys!

How can I refresh any page (table, view, form, relation manager or a page tab) after modal action performed?

In my case I on the ViewUser page in a RelationManager tab and I call header action of the page. After that I need to refresh relation manager table.

Thank you!

**Comment by leandrocfe** [comment]
Use livewire events

YourRelationManager.php
```php
use Livewire\Attributes\On;

#[On('updateRelationManagerPage')]
public function updateRelationManagerPage(): void
{
    //...
}
```
Your Action
```php
Actions\Action::make('updateRelationManager')
   ...
    ->after(fn () => $this->dispatch('updateRelationManagerPage')),
```
